### ISSUE TITLE
Is there any plans to support rpc api about token? 

### ISSUE THREAD

**Issue by yihau** [dfuse-io/solana-go#3]
I saw there are some api about token in [here](https://docs.solana.com/developing/clients/jsonrpc-api#unstable-methods).
Is these support in the future?

**Comment by maoueh** [comment]
Yes we could integrate the new APIs no problem, which method are you looking to use exactly? 

Note that new methods that exists but are not in library can still be used in your call, the lower level building blocks to call the RPC are now available via the `DoRequest` method.
